test/integration/kwild_test.go

```diff
@@ -307,52 +309,62 @@ func TestKwildEthDepositOracleIntegration(t *testing.T) {
 		t.Parallel()
 	}
 
-	opts := []integration.HelperOpt{
-		integration.WithBlockInterval(time.Second),
-		integration.WithValidators(4),
-		integration.WithNonValidators(0),
-		integration.WithGanache(),
-		integration.WithGas(),
-		integration.WithEthDepositOracle(true),
+	type testcase struct {
+		numValidators int
+		serviceNames  []string
+	}
+
+	testcases := []testcase{
+		{1, singleNodeServices},
+		{4, allServices},
 	}
 
 	testDrivers := strings.Split(*drivers, ",")
 	for _, driverType := range testDrivers {
-		t.Run(driverType+"_driver", func(t *testing.T) {
-			ctx := context.Background()
-
-			helper := integration.NewIntHelper(t, opts...)
-			helper.Setup(ctx, allServices)
-
-			// get deployer
-			ctxMiner, cancel := context.WithCancel(ctx)
-			defer cancel()
-			deployer := helper.EthDeployer(false)
-			err := deployer.KeepMining(ctxMiner)
-			require.NoError(t, err)
-
-			// Get the user driver
-			userDriver := helper.GetUserDriver(ctx, "node0", driverType, deployer)
-
-			// Node0 Driver
-			// node0Driver := helper.GetOperatorDriver(ctx, "node0", driverType)
-
-			// Approve the deposit
-			// specifications.ApproveSpecification(ctx, t, userDriver)
-
-			// Deposit the amount to the escrow
-			amount := big.NewInt(10)
-			specifications.DepositSuccessSpecification(ctx, t, userDriver, amount)
-
-			// Deposit Failure
-			// specifications.DepositFailSpecification(ctx, t, userDriver)
-
-			// Deploy DB without enough funds
-			specifications.DeployDbInsufficientFundsSpecification(ctx, t, userDriver)
-
-			// Deploy DB with enough funds
-			specifications.DeployDbSuccessSpecification(ctx, t, userDriver)
-		})
+		for _, tc := range testcases {
+			t.Run(driverType+"_driver", func(t *testing.T) {
```

Comment:
maybe modify test case name here? otherwise you dont' know which case fail(if any)